app/Http/routes.php

```diff
@@ -52,7 +52,9 @@
     // Necessary because IDK and please let's fix this @todo
     Route::post('talks/{id}', 'TalksController@update');
     Route::post('conferences/{id}', 'ConferencesController@update');
+    Route::post('bios/{id}', 'BiosController@update');
```

Comment:
Yah, that's why I was confused when I first wrote this. I didn't know why the Laravel HTML Form generator was submitting as POST and PUT and I didn't feel like fighting it. But yes, we need to fix/figure that out. I'll try to dig into it later.